File: CF/B_Long_Long.py
for _ in range(int(input())):
    n = int(input())

    arr = list(map(int,input().split(" ")))

    groups = []

    i = 0

    while i < n:

        if(arr[i] <= 0):
            add = []

            for j in range(i,n):
                if(arr[j] <= 0):
                    add.append(arr[j])
                else:
                    i = j+1
                    break
                if(j == n-1):
                    i = j+1
                    break
            
            groups.append(add)
        
        else:
            i += 1
    
    ans = 0
    for i in groups:
        for j in i:
            if j < 0:
                ans += 1
                break

    s = 0

    for i in arr:
        s += abs(i)

    print(s,ans)       

    
                

File: CF/B_Long_Long.cpp
#include <bits/stdc++.h>
#include <iostream>

using namespace std;

int main(){
    int tt; cin >> tt;

    while(tt--){
        int n; cin >> n;
        

        vector<int> arr(n);

        for(int i = 0; i < n; i++){
            cin >> arr[i];
        }

        vector<vector<int>> groups = vector<vector<int>>();

        int i = 0;

        while(i < n){

            if(arr[i] <= 0){
                vector<int> e = vector<int>();
                for(int j = i; j < n; j++){
                    if(arr[j] <= 0){
                        e.push_back(arr[j]);
                    }
                    else{
                        i = j+1;
                        break;
                    }
                }

                groups.push_back(e);
            }
            else{
                i++;
            }
            
            
        }

        int ans = 0;

        for(vector<int> i: groups){
            for(int j:i){
                if(j < 0){
                    ans++;
                    break;
                }
            }
        }

        cout << ans << "\n";


       
    }
}